src/screens/repo/screens/build/menu.js

```diff
@@ -115,25 +116,25 @@ export default class BuildMenu extends Component {
 								)}
 							</li>
 							<li>
-								{build.status === "success" ? (
+								{build.status === STATUS_SUCCESS  ? (
 									<button onClick={this.togglePromote}>
 										<DeployIcon />
 										<span>Promote Build</span>
 									</button>
 								):null }
+                                {build.status === STATUS_SUCCESS  && this.state.togglePromote ? (
```

Comment:
You are mixing TAB & spaces, coding standards used in drone-ui need to use tab, can you run linter  (execute `yarn lint` command from project root) and fix errors? Thx!